### PR TITLE
Add updated expo dependencies and config workspaces (WIP)

### PR DESCRIPTION
**Pull request by lnmunhoz**
I have to do some final tests. Will update this PR soon.

### CODE REVIEW

**Comment by Fried-Chicken** [comment]
Hey cool, thanks for raising this.

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   "license": "MIT",
   "dependencies": {
     "@expo/samples": "2.1.1",
-    "expo": "^30.0.1",
+    "expo": "32.0.6",
+    "metro-config": "^0.53.1",
     "react": "16.3.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
```
Comment (Fried-Chicken) [inline package.json]: Hey @lnmunhoz I think the react-native package would also need updating here.

Comment (lnmunhoz) [reply package.json]: @Fried-Chicken I fixed the dependencies. The app seems to be running now on IOS.

**Comment by lnmunhoz** [comment]
@Fried-Chicken Can you test on Android? IOS looks good now, the last commit fixed some warnings and the icons were updated :)

**Comment by Fried-Chicken** [comment]
@lnmunhoz Thanks for the great update. Tested and working so going to merge this! Great job! 👍🏻

**Pull request merged**